velox/vector/VectorStream.h

```diff
@@ -43,6 +43,19 @@ class VectorSerializer {
   /// Serialize all rows in a vector.
   void append(const RowVectorPtr& vector);
 
+  /// Usage
```

Comment:
It is nice to include an example of usage, but could you move it after the description?